src/app/page/Recipient/MissionFeedback.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useContext, useEffect } from "react";
+import { Mission } from "../../model";
+import { MissionStatus } from "../../model/schema";
+import Snackbar from "../../component/Snackbars";
+import {
+  DeliveryConfirmation,
+  Feedback,
+  FeedbackWithSuccess,
+  Acknowledgement,
+  Step,
+} from "./Steps";
+import { makeStyles, Slide, Modal, Backdrop, Paper, IconButton } from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    overflow: "auto",
+    paddingTop: "5rem",
+    paddingBottom: "2rem",
+    width: "100%",
+    maxWidth: theme.breakpoints.width("sm"),
+    position: "relative",
+    height: "100%",
+  },
+  modal: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    height: "100vh",
+  },
+  close: {
+    position: "absolute",
+    top: "0",
+    right: "0",
+    "& svg": {
+      width: "2.5rem",
+      height: "2.5rem",
+    },
+  },
+}));
+
+const MissionFeedback = ({ deliveryImage, missionUid, setShow, show }) => {
```

Comment:
This component was moved from `src/app/page/MissionFeedback/MissionFeedback.jsx`

Main changes here are updating the way we submit and wrapping everything in the modal and slide components